### ISSUE TITLE
omf/thefuck causes install error

### ISSUE THREAD

**Issue by bryaan** [jorgebucaran/fisher#408]
```fish
 fisher omf/thefuck
Installing 1 plugin/s
OK Copy thefuck
Unsupported use of '&&'. In fish, please use 'COMMAND; and COMMAND'.
- (line 1): begin; ;            __tf_alias () {;                TF_PYTHONIOENCODING=$PYTHONIOENCODING;;                export TF_ALIAS=__tf_alias;;                export TF_SHELL_ALIASES=$(alias);;                export TF_HISTORY="$(fc -ln -10)";;                export PYTHONIOENCODING=utf-8;;                TF_CMD=$(;                    thefuck THEFUCK_ARGUMENT_PLACEHOLDER $@;                ) && eval $TF_CMD;;                unset TF_HISTORY;;                export PYTHONIOENCODING=$TF_PYTHONIOENCODING;;                test -n "$TF_CMD" && print -s $TF_CMD;            };        ; 
                                                                                                                                                                                                                                                                                                                                                                                                                ^
from sourcing file -
	called on line 60 of file /usr/share/fish/functions/eval.fish

in function “eval”
	called on line 3 of file ~/.config/fish/conf.d/thefuck.init.fish

from sourcing file ~/.config/fish/conf.d/thefuck.init.fish
	called on line 805 of file ~/.config/fish/functions/fisher.fish

in function “__fisher_plugin_enable”
	called on line 445 of file ~/.config/fish/functions/fisher.fish
	with parameter list “/home/bryan/.config/fisherman/thefuck”

in function “__fisher_install”
	called on line 274 of file ~/.config/fish/functions/fisher.fish
	with parameter list “oh-my-fish/plugin-thefuck”

in function “fisher”
	called on standard input
	with parameter list “omf/thefuck”

Done in 40ms
```

And also when starting a new fish shell with plugin installed:

```fish
Unsupported use of '&&'. In fish, please use 'COMMAND; and COMMAND'.
- (line 1): begin; ;            __tf_alias () {;                TF_PYTHONIOENCODING=$PYTHONIOENCODING;;                export TF_ALIAS=__tf_alias;;                export TF_SHELL_ALIASES=$(alias);;                export TF_HISTORY="$(fc -ln -10)";;                export PYTHONIOENCODING=utf-8;;                TF_CMD=$(;                    thefuck THEFUCK_ARGUMENT_PLACEHOLDER $@;                ) && eval $TF_CMD;;                unset TF_HISTORY;;                export PYTHONIOENCODING=$TF_PYTHONIOENCODING;;                test -n "$TF_CMD" && print -s $TF_CMD;            };        ; 
                                                                                                                                                                                                                                                                                                                                                                                                                ^
from sourcing file -
	called on line 60 of file /usr/share/fish/functions/eval.fish

in function “eval”
	called on line 3 of file ~/.config/fish/conf.d/thefuck.init.fish

from sourcing file ~/.config/fish/conf.d/thefuck.init.fish
	called on line 207 of file /usr/share/fish/config.fish

from sourcing file /usr/share/fish/config.fish
	called during startup

source: Error while reading file “-”
```

Looks like that shouldn't be able to happen right?

**Comment by JorgeBucaran** [comment]
This looks like a mistake on your end. Some env variable that is causing [omf-thefuck](https://github.com/oh-my-fish/plugin-thefuck/blob/master/functions/__tf_updt.fish#L6) to pick up some non-fish code, etc.

**Comment by jethrokuan** [comment]
Some time ago, someone asked me why the `thefuck` plugin didn't work from omf, and I had to fork it here: https://github.com/jethrokuan/thefuck

can't remember the context and reason though.

**Comment by JorgeBucaran** [comment]
:) @jethrokuan If your plugin works, then I think we can close here.

**Comment by bryaan** [comment]
@jethrokuan So there is a version mismatch?

**Comment by JorgeBucaran** [comment]
What is `TF_PYTHONIOENCODING=$PYTHONIOENCODING;;`?

**Comment by JorgeBucaran** [comment]
https://github.com/oh-my-fish/plugin-thefuck/blob/c2701e037c5a721ac651de0fda00c67ca67ef259/init.fish#L3

→

https://github.com/oh-my-fish/plugin-thefuck/blob/c2701e037c5a721ac651de0fda00c67ca67ef259/functions/__tf_updt.fish#L6

→

？

**Comment by jethrokuan** [comment]
> @jethrokuan So there is a version mismatch?

I don't use `thefuck` personally, it was a request, here: https://github.com/fisherman/fzf/issues/35

I wouldn't call it a version mismatch, I think it just didn't work well with anything other than omf at that time, not sure about now.

**Comment by jorgebucaran** [comment]
I just released [fisher/releases/tag/3.0.0](https://github.com/jorgebucaran/fisher/releases/tag/3.0.0)~! 🎉

V3 maintains OMF package compatibility, but if it still doesn't work feel free to open a new issue. Keep in mind, however, that I have no intention to support every little OMF idiosyncrasy.

---

For more information see the [new documentation](https://github.com/jorgebucaran/fisher/blob/34b5257c5beaaf37f7d1988a6a9a62f8a0b1196c/README.md).
